shared/src/main/scala/mlscript/syntax.scala

```diff
@@ -122,11 +122,11 @@ trait IdentifiedTerm
 sealed abstract class SimpleTerm extends Term with IdentifiedTerm with SimpleTermImpl
 
 sealed trait Statement extends StatementImpl
-final case class LetS(isRec: Bool, pat: Term, rhs: Term)  extends Statement
-final case class DataDefn(body: Term)                     extends Statement
-final case class DatatypeDefn(head: Term, body: Term)     extends Statement
-final case class Constructor(params: Tup, body: Blk)      extends Statement // constructor(...) { ... }
-class Import(val path: Str)                               extends Statement
+final case class LetS(isRec: Bool, pat: Term, rhs: Term) extends Statement
+final case class DataDefn(body: Term)                    extends Statement
+final case class DatatypeDefn(head: Term, body: Term)    extends Statement
+final case class Constructor(params: Tup, body: Blk)     extends Statement // constructor(...) { ... }
+class Import(val path: Str)                              extends Statement
```

Comment:
Not sure about these changes...

Reply:
Oh. I found I misaligned the original code in the previous commits, so I undid it in the refactor.